WordPress/Classes/Services/CommentService.m

```diff
@@ -1195,6 +1195,13 @@ - (NSArray *)topLevelCommentsForPage:(NSUInteger)page forPost:(ReaderPost *)post
     return fetchedObjects;
 }
 
+- (NSArray *)topLevelComments:(NSUInteger)number forPost:(ReaderPost *)post
+{
+    NSArray *comments = [self topLevelCommentsForPage:1 forPost:post];
```

Comment:
I know this is out of scope and we're only going to pass in `2` here for the comments below post; But since this is a public method, the implementation could be misleading as it only returns comments from page 1. 

Should the method also fetch comments from page 2 (or more) in case if there are comments on the next pages and the `number` is not yet fulfilled?

Reply:
Since there are existing methods to get more comments with correct pagination, I purposefully restricted the new ones to page 1 since they are intended to get a minimal number of comments. But you make a good point - it is not clear. So instead of expanding them to allow multiple pages, I added comments noting that they are intentionally restricted to page 1.